openedx2zim/utils.py

```diff
@@ -158,3 +158,20 @@ def get_response_headers(url):
         if response_headers.get("content-length", None) is not None:
             return response_headers["content-length"], content_ext
     return None, content_ext
+
+
+def remove_autogenerated_tags(html_string):
+    """ removes <head> and <html> and <body> tags from endpoints of a string """
+    if html_string.startswith("<html>"):
```

Comment:
You can make it a bit more flexible with
```py
    html_string = html_string.strip()
    for tag in ("html", "head", "body"):
        search_string = f"<{tag}>"
        if html_string.startswith(search_string):
            html_string = html_string[len(search_string):].strip()
        search_string = f"</{tag}>"
        if html_string.endswith(search_string):
            html_string = html_string[:-len(search_string)].strip()
    return html_string
```

Reply:
Oh yup. I initially thought I'll simplify that, but forgot. Thanks. Did that.